services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -448,3 +448,4 @@ async def get_task_output(
         comp_task = await result.fetchone()
         if comp_task:
             return comp_task.outputs
+        return None
```

Comment:
well not sure I get the use of adding this. the return type is already marked as Optional

Reply:
you are totally right